### PR TITLE
Added Bower sample. Closes #24.

### LINKED ISSUE #24: Need a bower sample
We could just use postinstall to fire off bower install, or we could include this in the gulp sample

### PR DESCRIPTION
**Pull request by jmdobry**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,4 @@ npm-debug.log
 coverage/
 
 test/encrypted/nodejs-docs-samples.json
+dump.rdb
```
Comment (jonparrott) [inline .gitignore]: What is this?

**Comment by jonparrott** [comment]
LGTM.

**Comment by JustinBeckwith** [comment]
LGTM

File: .gitignore (hunk 1 above)
Comment (jmdobry) [inline .gitignore]: Something created by redis-server when I run tests locally. It was bugging me, and I didn't want to accidentally commit it.

**Pull request merged**